src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/XamarinProject.cs

```diff
@@ -272,7 +272,7 @@ public virtual void UpdateProjectFiles (string directory, IEnumerable<ProjectRes
 					continue;
 				}
 				string filedir = directory;
-				if (path.Contains (Path.DirectorySeparatorChar)) {
+				if (path.Contains ($"{Path.DirectorySeparatorChar}")) {
```

Comment:
@jonathanpeppers do we prefer this or `.ToString` ?

Reply:
I think both *look* fine, so I looked at the IL produced:
```msil
// if (text.Contains($"{Path.DirectorySeparatorChar}"))
IL_00bb: ldloc.2
IL_00bc: ldstr "{0}"
IL_00c1: ldsfld char [netstandard]System.IO.Path::DirectorySeparatorChar
IL_00c6: box [netstandard]System.Char
IL_00cb: call string [netstandard]System.String::Format(string, object)
IL_00d0: callvirt instance bool [netstandard]System.String::Contains(string)
```
So I think it only matters if perf is a concern, `ToString()` would be better and avoid a `string.Format("{0}", Path.DirectorySeparatorChar)`.

This PR is green and it's a test, so I wouldn't change it.

Reply:
Its good to know that `ToString` is more performant :D